runtime/oti/vm_api.h

```diff
@@ -4345,14 +4345,14 @@ jint
 isPinnedContinuation(J9VMThread *currentThread);
 
 /**
- * @brief  Walk the stackframes associated with Continuation object
+ * @brief Walk the stackframes associated with a continuation
  *
- * @param currentThread
- * @param continuationObject
- * @return UDATA
+ * @param currentThread current thread
+ * @param continuation the continuation to be walked
+ * @return UDATA a JVMTI error code
```

Comment:
`walkContinuationStackFrames` does not return a JVMTI error code
```suggestion
 * @return 0 on success and non-zero on failure
```